src/modules/account/components/AccountTile.vue

```diff
@@ -195,11 +195,34 @@ export default{
         return
       }
     }
+
+    const setDefaultAccInStorage = (address) =>{
+      let data = [{
+            walletName: walletState.currentLoggedInWallet.name,
+            defaultAcc: address
+          }]
```

Comment:
why it is in array ? it does not need to use array, just object

Reply:
Noted

Reply:
Noted